src/app.js

```diff
@@ -1 +1,65 @@
-'use strict';
+/* eslint-disable no-console */
+import fs from 'fs/promises';
+
+const sourceFilePath = process.argv[2];
+const destinationFilePath = process.argv[3];
+
+const isDirectory = async (path) => {
+  try {
+    const stats = await fs.stat(path);
+
+    return stats.isDirectory();
+  } catch (error) {
+    console.error(error);
```

Comment:
Consider providing a more descriptive error message here to help with debugging.